dev-local/darwin-init.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
@vuksa, we have update-sdk script now. It can probably be shared between Mac and Linux